packages/button/__tests__/Button.test.tsx

```diff
@@ -83,22 +82,19 @@ describe('with props.size', () => {
 })
 
 describe('with props.variant', () => {
-  it.each<Required<ButtonProps<'button'>>['variant']>([
-    'primary',
```

Comment:
Were `overlay` and `critical` removed from the variants?

Reply:
We never had `overlay` and `critical` here